packages/components/src/internal/components/search/utils.ts

```
@@ -1044,3 +1038,29 @@ export function getSearchScopeFromContainerFilter(cf: Query.ContainerFilter): Se
             return SearchScope.FolderAndSubfolders;
     }
 }
+
+export function getSampleFinderTabRowCountSql(queryModel: QueryModel): string {
+    const filters = queryModel.baseFilters;
+    const wheres = [];
+    filters.forEach(filter => {
+        let clause = '';
+        if (filter.getFilterType().getURLSuffix() === COLUMN_NOT_IN_FILTER_TYPE.getURLSuffix()) {
+            clause = 'RowId NOT IN (' + filter.getValue() + ')';
+        } else if (filter.getFilterType().getURLSuffix() === COLUMN_IN_FILTER_TYPE.getURLSuffix()) {
+            clause = 'RowId IN (' + filter.getValue() + ')';
+        } else if (filter.getFilterType().getURLSuffix() === IN_EXP_DESCENDANTS_OF_FILTER_TYPE.getURLSuffix()) {
+            clause = 'expObject() IN EXPDESCENDANTSOF (' + filter.getValue() + ')';
+        } else {
+            console.error('Bad filter');
```

Should this throw an error instead of logging an error?

---

I'm working on a story to support more filter card/types for sample finder, after which all filter types are possible. I'll just leave this console.error as is for now.